web/src/lib/components/asset-viewer/asset-viewer.svelte

```diff
@@ -103,6 +104,8 @@
       $stackAssetsStore = [...$stackAssetsStore, asset].sort(
         (a, b) => new Date(b.fileCreatedAt).getTime() - new Date(a.fileCreatedAt).getTime(),
       );
+      // if its a stack, add the next stack image in addition to the next asset
+      preloadAssets.push($stackAssetsStore[1]);
```

Comment:
It's technically possible to create a stack with only one asset, opening such a stack will show the spinner forever

Reply:
Good to know, I don't have any native stacked photos and was only able to create a stack with 2+ photos in the UI